scripts/apps/authoring-react/article-widgets/translations/translations.tsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import {RelativeDate} from 'core/datetime/relativeDate';
+import {state as State} from 'apps/search/components/fields/state';
+import {IArticle, IArticleSideWidget, IExtensionActivationResult, IRestApiResponse} from 'superdesk-api';
+import {gettext} from 'core/utils';
+import {httpRequestJsonLocal} from 'core/helpers/network';
+import {openArticle} from 'core/get-superdesk-api-implementation';
+import {AuthoringWidgetLayout} from 'apps/dashboard/widget-layout';
+import {AuthoringWidgetHeading} from 'apps/dashboard/widget-heading';
+import {Card} from 'core/ui/components/Card';
+import {Spacer, SpacerBlock} from 'core/ui/components/Spacer';
+import {Label} from 'superdesk-ui-framework';
+
+const getLabel = () => gettext('Translations');
+
+type IProps = React.ComponentProps<
+    IExtensionActivationResult['contributions']['authoringSideWidgets'][0]['component']
+>;
+
+interface IState {
+    translations: Array<IArticle>;
+    translationsLookup: Dictionary<IArticle['_id'], IArticle>;
+}
+
+class Translations extends React.Component<IProps, IState> {
+    constructor(props: IProps) {
+        super(props);
+
+        this.state = {
+            translations: null,
+            translationsLookup: null,
+        };
+    }
+
+    getTranslations(): Promise<IRestApiResponse<IArticle>> {
+        return httpRequestJsonLocal<IRestApiResponse<IArticle>>({
```

Comment:
try making this use `ISuperdeskQuery` [\[example\]](https://github.com/superdesk/superdesk-client-core/blob/broadcasting/scripts/api/article-send.tsx#L20)